bindings/nodejs/lib/types/block/core/basic.ts

```diff
@@ -0,0 +1,37 @@
+// Copyright 2023 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+import { Payload, PayloadDiscriminator } from '../payload';
+import { Type } from 'class-transformer';
+import { StrongParents, WeakParents, ShallowLikeParents } from '../parents';
+import { Block, BlockType } from '.';
+
+/**
+ * Basic Block layout.
+ */
+export class BasicBlock extends Block {
+    /**
+     * Blocks that are strongly directly approved, in the range of 1..8.
```

Comment:
Where did you take these comments? REST TIP? These ranges sound like a headache to maintain if they change, I wouldn't mention them.

Reply:
Its like how we want to do the rust parents per block type. However as we dont do validation on nodejs i left it to the comment. This was taken from our rust code a few weeks back.
I think its good to know what to expect, and know that an empty array is fine. 
I could switch it to "this array can be empty" for weak and shallow, and "cannot be" for strong?

Reply:
I don't know, even that sounds like too much information. People are supposed to use high level functions we provide so they don't care about validation rules.

Reply:
Well good thing "Blocks that are strongly directly approved" explains it all :trollface: ive removed the numbers